### DISCUSSION TITLE
v2.11.2: renewal of some of the SSL certs are returning internal error

### DISCUSSION THREAD

**Discussion by stefanneuweger** [NginxProxyManager/nginx-proxy-manager#3751] [General]
I am running the latest version of npm.
Since SSL certs for my 5 proxy hosts are expiring, i am looking to renew them in npm.
I was able to renew 2 of those 5.
For 3 of them npm raises an "internal error" message. 
I researched here in the discussions but wasn't able to find a solution approach for my scenario.
Can you guys please advise as to what could be the potential spots to check?
Obviously connection etc. is working, also letsencrypt limits shouldn't be an issue with the low number of ssl renewal .
Strange thing to me is that it works for some but for others not.
thanks for your support.

**Comment by stefanneuweger** [comment]
as a workaround i now deleted the respective ssl certs and also the proxy hosts completey and entered them from scratch with requesting a new ssl cert which did work.

**Comment by boehamian** [comment]
Getting same error. Have attempted to use different versions with no success. Renewing certificates did not work either.
